src/main/java/seedu/commands/currency/CurrencyConversionCommand.java

```diff
@@ -0,0 +1,29 @@
+package seedu.commands.currency;
+
+import seedu.commands.Command;
+import seedu.exceptions.SameCurrencyTypeException;
+import seedu.utility.BudgetManager;
+import seedu.utility.CurrencyManager;
+import seedu.utility.FinancialTracker;
+import seedu.utility.Ui;
+
+public class CurrencyConversionCommand extends Command {
+    protected CurrencyType from = CurrencyType.SGD;
```

Comment:
`from` cannot be SGD by default right? How do you change back?

Reply:
For now, it works fine since there are only two currency modes. I think you're right, I have to change later if I accommodate more currency types.

Reply:
I'm going to add Indian Rupee first in my next PR, I'll fix this issue in that.